roles/tpa_single_node/tasks/podman.yml

```diff
@@ -3,7 +3,7 @@
   when:
     - ansible_facts['distribution'] == 'RedHat'
     - ansible_facts['distribution_major_version'] == '9'
-  ansible.builtin.command: podman login registry.redhat.io -u {{ tpa_single_node_registry_username }} --password {{ tpa_single_node_registry_password }}
+  ansible.builtin.command: podman login registry.redhat.io -u {{ tpa_single_node_registry_username }} -p {{ tpa_single_node_registry_password }}
```

Comment:
Is there a danger of leaking password to the terminal/logs?

Reply:
I believe so when using Ansible along those type of command.
Meanwhile this better to have secret values in envar than in plain text inside configuration files !
This is why using an auth access token (see description) is always safer.